net.pioneerspacesim.Pioneer.json

```diff
@@ -0,0 +1,102 @@
+{
+	"app-id": "net.pioneerspacesim.Pioneer",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "1.6",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "pioneer",
+
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--device=dri",
+		"--persist=.pioneer"
+	],
+
+	"build-options" : {
+		"cflags": "-O3",
+		"cxxflags": "-O3",
+		"strip": true,
+		"env": {
+			"PIONEER_DATA_DIR": "/app/usr/share/pioneer"
```

Comment:
It's needed only during build or also for running?

Reply:
Only for building (then the path is saved within the executable).

Reply:
:+1: